src/AspNet.Security.OAuth.Alipay/AlipayAuthenticationHandler.cs

```diff
@@ -213,5 +213,40 @@ private string GetRSA2Signature([NotNull] SortedDictionary<string, string> sorte
 
             return Convert.ToBase64String(encryptedBytes);
         }
+
+         /// <inheritdoc />
+        protected override string BuildChallengeUrl(AuthenticationProperties properties, string redirectUri)
+        {
+            var scopeParameter = properties.GetParameter<ICollection<string>>(OAuthChallengeProperties.ScopeKey);
+            var scope = scopeParameter != null ? FormatScope(scopeParameter) : FormatScope();
+
+            var parameters = new Dictionary<string, string>
+            {
+                { "app_id", Options.ClientId }, // Replace "client_id"
+                { "scope", scope },
+                { "response_type", "code" },
+                { "redirect_uri", redirectUri },
```

Comment:
```suggestion
                ["app_id"] = Options.ClientId, // Used instead of "client_id"
                ["scope"] = scope,
                ["response_type"] = "code",
                ["redirect_uri"] = redirectUri,
```